watertap/costing/util.py

```diff
@@ -100,7 +100,7 @@ def cost_membrane(blk, membrane_cost, factor_membrane_replacement):
     )
 
 
-def cost_rectifier(blk, ac_dc_conversion_efficiency=0.90):
+def cost_rectifier(blk, power=100 * pyo.units.kW, ac_dc_conversion_efficiency=0.90):
```

Comment:
do you mean adding a warning that user did not provide power variable to the unit? I think that should definetly be a warning (I assume it should throw an error, as it won't find a power value).

Reply:
Honestly,  I think this is ok and we should assume the user has basic abilities to avoid mistakes from taking in default values.  We assign default values to variables all over the places. If we worry about this, we probably should not give default values to anything that is subjective to change.  Also at this point, the use of the `util` will also pair with a unit having a `power` and the chance of mistake is even lower.    I would leave as-is  now.

Reply:
The difference is in other locations, using the default value for a variable is assuming that the variable equals that default value. Its like the conversion efficiency here. Not changing from the default is simply accepting 0.9 as the value. Using the 'default' for the power variable here is direct misuse of the function in most if not all applications of the function, which I why I think it warrants a warning.